src/clients/storage/CassandraTableStorageManager.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+let asyncEachLimit = require('async/eachLimit');
+
+/** http://docs.datastax.com/en/developer/nodejs-driver/3.2/features/batch/
+ * TODO: ASYNC_BATCH_LIMIT will need to be tuned.
+ * It is best to keep the batch size small (in the order of 10s).
+ * If the batch size is greater than 5k, then the server will issue a warning.
+ */
+const ASYNC_BATCH_LIMIT = 10;
+
+module.exports = {
```

Comment:
In what way does this class add value on top of the CassandraClient? Would it make sense for the CassandraClient to abstract away/hide the batching logic so that it's easier for other code to use the client (i.e., they'd only have to know about one class, not multiples).